pyproject.toml

```diff
@@ -22,18 +30,19 @@ classifiers = [
     "Topic :: Software Development :: Libraries :: Python Modules",
     "Topic :: Software Development :: Quality Assurance",
 ]
+urls = { Documentation = "https://github.com/life4/deal" }
 
-[tool.flit.metadata.requires-extra]
+[project.optional-dependencies]
 all = [
-    "astroid>=2.11.0",    # for tyupe inference in linter
-    "deal-solver>=0.1.2", # for formal verification
```

Comment:
Is there a reason fo removing the comments? We're very rigorous about dependencies we bring into the project, and the comments are intended to identify why a particular dependency is here.